prow/jobs.md

```diff
@@ -118,9 +118,8 @@ to Prow in some other context, like `/close`. It is similarly not suggested to d
 
 #### Posting GitHub Status Contexts
 
-Jobs that run will always post a status context to the commit under test in GitHub.
-Jobs that run conditionally but do not match the content of the pull request will
-_not_ post "Skipped" status contexts to the pull request.
+Presubmits that do not have `skip_report: true` will always post a status context to the commit under test in GitHub.
```

Comment:
nit: text wrapping length

Reply:
Technically now postsubmits can publish

Reply:
Could we flip the negative here to have the common case first? 

```
Jobs will post a status context in GitHub to the commit under test, unless the job is configured with `skip_report: true`
```

Reply:
Done.

The text wrapping still displays correctly if we don't manually manage the line length. I'm not a fan of shuffling paragraph text around when I add a few words to the first sentence so whenever I touch markdown text that manually manages line length I switch it to long lines and let GH manage the wrapping.